src/coreclr/gc/gcpriv.h

```diff
@@ -5334,6 +5407,10 @@ class gc_heap
     PER_HEAP_ISOLATED_FIELD_DIAG_ONLY uint64_t last_alloc_reset_suspended_end_time;
     PER_HEAP_ISOLATED_FIELD_DIAG_ONLY size_t max_peak_heap_size;
 
+    // Sometimes it's difficult to figure out why we get the gen0 min/max budget.
+    // These fields help figure those out.
+    PER_HEAP_ISOLATED_FIELD_DIAG_ONLY size_t llc_size;
```

Comment:
It appears to me that this field is possibly optimized away in Windows/x64/Release build because it is write-only. Apparently, that is not what we want.

Reply:
oh right, I made `bgc_init_n_heaps` volatile but forgot to make this volatile. thanks for noticing!